src/olympia/files/models.py

```diff
@@ -76,6 +76,14 @@ def files_storage():
     return SafeStorage(root_setting='ADDONS_PATH')
 
 
+class FileManager(ManagerBase):
+    def disabled_to_be_reenabled(self):
```

Comment:
Nit: I originally read this as `disabled_to_reenabled` and it made me think it was previously disabled and now enabled. Maybe something like `disabled_with_original_status`? Not sure if that reads differently to you

Reply:
I was aiming for it to be descriptive of the meaning - what the File instances are - rather than the state, but I'm also not happy with the the naming. Is `disabled_that_would_be_renabled_with_addon` too much of a mouthful?